README.md

```diff
@@ -6,24 +6,70 @@ For Angular 1 see [ng-redux](https://github.com/wbuchwalter/ng-redux)
 [![Circle CI](https://circleci.com/gh/angular-redux/ng2-redux/tree/master.svg?style=svg)](https://circleci.com/gh/angular-redux/ng2-redux/tree/master)
 [![npm version](https://img.shields.io/npm/v/ng2-redux.svg?style=flat-square)](https://www.npmjs.com/package/ng2-redux)
 
-`ng2-redux` lets you easily connect your Angular 2 components with Redux.
+## Why use Redux?
+
+Redux is a predictable state container for complex
+web applications. It uses one-way data-flow and global
+immutable state to make your application easier to reason
+about.
+
+Redux also has a healthy community of developer tools
+and middleware that can help simplify your application
+even further.
+
+## Why use Ng2-Redux?
+
+Redux itself is framework-agnostic. However there exist
+layers (called 'bindings') to provide integration points
+with various frameworks for a first-class experience.
+
+* For React, there's [react-redux](https://npmjs.com/package/react-redux).
+* For Angular 2, there's Ng2-Redux.
+
+Ng2-Redux provides several advantages to the Angular 2
+developer:
+
+1. It plays well with the Angular 2 dependency injector.
+2. It has first-class support for a TypeScript-based workflow, using
+the official Redux typings.
+3. It exposes state as RxJS Observables for
+better integration with Angular 2's API.
+4. It cuts down the boilerplate often associated
+with Redux using TypeScript decorators for a simple,
+declarative experience.
+
+Note that this is not a reimplementation of the Redux
+pattern. _ng2-redux __is__ Redux_. This means
```

Comment:
"_ng2-redux __is__ Redux_" is not, strictly speaking true. Maybe go with: "ng2-redux is what allows you to use the actual Redux library in an Angular 2 application".